docker-compose.yml

```diff
@@ -78,6 +80,25 @@ services:
       - agile-core:agile
     restart: always
 
+
+
+  agile-idm:
+    image: agileiot/agile-idm-armv7l
```

Comment:
It'd be nice to have versioned tag here but it's okay for now. (same for other images)